jdbc/src/java/org/apache/hive/jdbc/HiveConnection.java

```diff
@@ -896,39 +946,103 @@ private void openSession() throws SQLException {
       openReq.setPassword(sessConfMap.get(JdbcConnectionParams.AUTH_PASSWD));
     }
 
+    //TODO(Vihang): This is a bit hacky. We piggy back on a dummy OpenSession call
```

Comment:
TODO: If nothing to do, should we remove your name from the comment?

Reply:
yeah, sorry. I removed my name. I had it to track all the todos.